### PR TITLE
Update manpages

### PR DESCRIPTION
**Pull request by zklamm**
Remove unused section headers and add description to `deploy` man page.

### CODE REVIEW

File: doc/man/maestro-config.1 [hunk 1]
```diff
@@ -43,8 +43,6 @@ Since Maestro is in the prototypal phase, bug reports are welcome and can be sub
 the GitHub Issues page
 .UE .
 
-.SH EXAMPLE
-
```
Comment (johnisom) [inline doc/man/maestro-config.1:47]: Do you think we could just have a quick example written here and showing what the prompt would look like?

Comment (zklamm) [reply doc/man/maestro-config.1]: See latest commit.

File: doc/man/maestro-config.1 [hunk 2]
```diff
@@ -45,6 +45,42 @@ the GitHub Issues page
...
+
+Please enter your AWS Account Number [********XXXX]:
+
+$ XXXXXXXXXXXX
+
+Please enter the region for your AWS services [us-east-2]:
+
+$ us-east-2
+.RE
+
```
Comment (johnisom) [inline doc/man/maestro-config.1:83]: In accordance of man page "**man-pages**(7)" under "**STYLE GUIDE**" > "**Formatting conventions (general)**":

> When showing example shell sessions, user input should be formatted in bold

, we should change the user input to be bold. I can commit that change if you like.

Comment (zklamm) [reply doc/man/maestro-config.1:83]: Sounds good. Go for it.

**Pull request merged**